### ISSUE TITLE
Request for joining WG

### ISSUE THREAD

**Issue by NawafSwe** [nodejs/undici#919]
Hi maintainers, I hope you are doing great!. 
I really appreciate the hard work you are putting to develop nodejs community, I am passionate in nodejs and its community. 
How can I join this working group and be one of its collaborator.

**Comment by mcollina** [comment]
The best way to start is for you to start working on a few issues: https://github.com/nodejs/undici/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22.

**Comment by NawafSwe** [comment]
There is a security issue with one of the used packages inside undici.
```
lodash  <4.17.21
Severity: high
Command Injection - https://npmjs.com/advisories/1673
fix available via `npm audit fix`
node_modules/tap/node_modules/lodash

1 high severity vulnerability
```

**Comment by NawafSwe** [comment]
> There is a security issue with one of the used packages inside undici.
> 
> ```
> lodash  <4.17.21
> Severity: high
> Command Injection - https://npmjs.com/advisories/1673
> fix available via `npm audit fix`
> node_modules/tap/node_modules/lodash
> 
> 1 high severity vulnerability
> ```

it can be fixed by chaning the version to ^4.17.21 locally

**Comment by mcollina** [comment]
I'm not sure why you are writing those things here. FWIW none of those are relevant, tap is only our test runner.
